src/features/shields/structure.ts

```diff
@@ -0,0 +1,318 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from '../../theme'
+import palette from '../../theme/palette'
+import Background from './media/background.svg'
+import { BlockedInfoRowStats, BlockedInfoRowText } from './display'
+import { StyledWrapper as Toggle } from '../shields/toggle/style'
+
+/**
+ * Main wrapper
+ */
+export const ShieldsPanel = styled<{}, 'div'>('div')`
+  box-sizing: border-box;
+  -webkit-font-smoothing: antialiased;
+  font-family: ${p => p.theme.fontFamily.heading};
+  position: relative;
+  background: #FFFFFF;
+  height: 100%;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto auto auto auto;
+`
+
+interface ShieldsHeaderProps {
+  status: 'enabled' | 'disabled'
+}
+
+export const ShieldsHeader = styled<ShieldsHeaderProps, 'header'>('header')`
+  box-sizing: border-box;
+  background-image: url(${Background});
+  background-repeat: no-repeat;
+  background-position: ${p => p.status === 'enabled' ? '0 -15px' : '0 0'};
```

Comment:
100% personal preference, but thought this element was riding a little high:
`background-position: ${p => p.status === 'enabled' ? '0 -5px' : '0 0'};`

Reply:
sounds good but deferring to @karenkliu